src/pages/LoginAndRegister/GuestAccountClaim.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<system-page>
+		<div class="page-content">
+			<h1 class="featured-text">
+				One last thing!
+			</h1>
+			<p>
+				To finish creating your account, please enter your first and last name below.
+			</p>
+			<form id="guestAccountClaimForm" action="." @submit.prevent.stop="claimGuestAccount">
+				<label class="input-label" for="firstName">
+					First name
+				</label>
+				<input id="firstName" v-model="firstName" type="text" required @valid="firstName = $event">
```

Comment:
The `@valid="firstName = $event"` isn't needed because the name is already being updated by the `v-model` propertry.